android-components/components/support/ktx/src/test/java/mozilla/components/support/ktx/android/view/ActivityTest.kt

```diff
@@ -34,6 +35,7 @@ class ActivityTest {
 
     private lateinit var activity: Activity
     private lateinit var window: Window
+    private lateinit var layoutParams: WindowManager.LayoutParams
```

Comment:
`private lateinit var windowInsetsController: WindowInsetsController` & `windowInsetsController = mock()` can be removed now 🙂